src/main/java/pokecube/world/WorldgenTags.java

```diff
@@ -27,6 +27,9 @@ public static void initTags()
             Registry.CONFIGURED_STRUCTURE_FEATURE_REGISTRY,
             new ResourceLocation("pokecube_world", "mixin_restrictions/no_basalt"));
 
-    public static TagKey<Biome> IS_ERODED_BADLANDS = TagKey.create(Registry.BIOME_REGISTRY,
-            new ResourceLocation("pokecube_world", "is_custom_eroded_badlands"));
+    public static TagKey<Biome> IS_BADLANDS_PILLARS = TagKey.create(Registry.BIOME_REGISTRY,
```

Comment:
this should be `IS_ERODED_<name>`, changing `<name>` to `BIOME` might be fine, but it shouldn't be changed to pillars.